src/downloader.rs

```diff
@@ -71,6 +86,13 @@ impl Downloader {
         let status = res.status();
         summary = Summary::new(download.clone(), status, size);
 
+        // Create the progress bar.
+        let style = ProgressStyle::with_template(
+            "{bar:40.cyan/blue} {bytes}/{total_bytes} ({bytes_per_sec}, {eta})",
```

Comment:
does bar:40 means that the progress bar will be 40 characters wide?

Reply:
Correct! For v2, I plan on making it configurable, but for now I hardcode the configuration.

As an example, it looks like this:
<img width="543" alt="image" src="https://user-images.githubusercontent.com/6969134/160170746-497e2789-05c6-4f95-aabb-7c06e1dfb8b5.png">